packages/experiments/src/components/Card/CardItem/CardItem.types.ts

```diff
@@ -18,6 +18,21 @@ export interface ICardItemProps extends ICardItemSlots, IStyleableComponentProps
    * @defaultvalue false
    */
   disableChildPadding?: boolean;
+
+  /**
+   * Defines how much to grow the StackItem in proportion to its siblings.
+   */
+  grow?: boolean | number | 'inherit' | 'initial' | 'unset';
+
+  /**
+   * Defines at what ratio should the StackItem shrink to fit the available space.
+   */
+  shrink?: boolean | number | 'inherit' | 'initial' | 'unset';
+
+  /**
+   * Defines how to align the StackItem along the x-axis (for vertical Stacks) or the y-axis (for horizontal Stacks).
```

Comment:
I am not following... what `StackItem` has to do with the `CardItemProps`

Reply:
This is pretty much a WIP that we started today and we have to clean up moving forward. That said, `CardItem` shares many props with `StackItem`, to the point where we are thinking about extending `CardItem` from `StackItem`, though further discussion is needed.